circuits/sybil/sybilResCredentialAtomicQuerySigOffChain.circom

```diff
@@ -0,0 +1,250 @@
+pragma circom 2.0.0;
+
+include "sybilUtils.circom";
+
+template SybilResCredentialAtomicQuerySigOffChain(IssuerLevels, ClaimLevels, gistLevels) {
+    
+    /* userID ownership signals */
+    signal input userGenesisID;
+    signal input profileNonce; /* random number */
+    signal input claimSubjectProfileNonce; // nonce of the profile that claim is issued to, 0 if claim is issued to genesisID
+
+    signal input issuerID;
+
+    // issuer auth proof of existence
+    signal input issuerAuthClaim[8];
+    signal input issuerAuthClaimMtp[IssuerLevels];
+    signal input issuerAuthClaimsRoot;
+    signal input issuerAuthRevRoot;
+    signal input issuerAuthRootsRoot;
+    signal output issuerAuthState;
+
+    // issuer auth claim non rev proof
+    signal input issuerAuthClaimNonRevMtp[IssuerLevels];
+    signal input issuerAuthClaimNonRevMtpNoAux;
+    signal input issuerAuthClaimNonRevMtpAuxHi;
+    signal input issuerAuthClaimNonRevMtpAuxHv;
+
+    // claim issued by issuer to the user
+    signal input issuerClaim[8];
+    // issuerClaim non rev inputs
+    signal input issuerClaimNonRevMtp[IssuerLevels];
+    signal input issuerClaimNonRevMtpNoAux;
+    signal input issuerClaimNonRevMtpAuxHi;
+    signal input issuerClaimNonRevMtpAuxHv;
+    signal input issuerClaimNonRevClaimsRoot;
+    signal input issuerClaimNonRevRevRoot;
+    signal input issuerClaimNonRevRootsRoot;
+    signal input issuerClaimNonRevState;
+
+    // issuerClaim signature
+    signal input issuerClaimSignatureR8x;
+    signal input issuerClaimSignatureR8y;
+    signal input issuerClaimSignatureS;
+
+    /** Query */
+    signal input issuerClaimSchema;
+
+    signal input claimPathNotExists; // 0 for inclusion, 1 for non-inclusion
+    signal input claimPathMtp[ClaimLevels];
+    signal input claimPathMtpNoAux; // 1 if aux node is empty, 0 if non-empty or for inclusion proofs
+    signal input claimPathMtpAuxHi; // 0 for inclusion proof
+    signal input claimPathMtpAuxHv; // 0 for inclusion proof
+    signal input claimPathKey; // hash of path in merklized json-ld document
+    signal input claimPathValue; // value in this path in merklized json-ld document
```

Comment:
This signals are something you don't need at all

Reply:
Done